draft-ietf-oauth-device-flow.xml

```diff
@@ -415,33 +468,169 @@
   <t hangText="authorization_pending"> <vspace blankLines="1"/>
 
            The authorization request is still pending as the end-user
-           hasn't yet visited the authorization server and entered their
-           verification code.</t>
+           hasn't yet completed the User Interaction steps (<xref target="UserInstruction" />). The client should repeat
+           the Access Token Request to the token endpoint.</t>
 
   <t hangText="slow_down"> <vspace blankLines="1"/>
            The client is polling too quickly and should back off at a
            reasonable rate.</t>
 
   <t hangText="expired_token"> <vspace blankLines="1"/>
-           The device_code has expired. The client will need to make a new
+           The <spanx style="verb">device_code</spanx> has expired. The client will need to make a new
            Device Authorization Request.</t>
 
   </list> 
   </t>
 
-  <t>The error code "authorization_pending" and "slow_down" are considered
-    soft errors. The the client should continue to poll when
-    receiving "authorization_pending"
-    errors, reducing the interval if a "slow_down" error is received.
-    Other error codes are considered hard errors, the client should stop polling
-    and react accordingly, for example, by displaying an error to the user.
+  <t>The error codes <spanx style="verb">authorization_pending</spanx> and
+    <spanx style="verb">slow_down</spanx> are considered soft errors. The client
+    should continue to poll the token endpoint by repeating the Device
+    Token Request (<xref target="TokenRequest" />) when receiving soft errors,
+    increasing the poll interval if a <spanx style="verb">slow_down</spanx> error
+    is received. Other error codes are considered hard errors, the client should
+    stop polling and react accordingly, for example, by displaying an error to
+    the user.
+  </t>
+
+  <t>
+    The interval at which the client polls MUST NOT be more frequent than the
+    <spanx style="verb">interval</spanx> parameter returned in the Device Authorization
+    Response (see <xref target="AuthorizationResponse"/>).
+  </t>
+
+  <t>
+  The assumption of this specification is that the secondary device the
+  user is authorizing the request on does not have a way to communicate back
+  to the OAuth client. That only a one-way channel is required makes this flow
```

Comment:
s/That only a one-way channel is required makes this flow useful in many scenarios such as a HTML application on TV that can only make outbound requests.
/Only a one-way channel is required to make this flow useful in many scenarios. For example, an HTML application on a TV that can only make outbound requests.